### ISSUE TITLE
Security issue in jsdom dependency

### ISSUE THREAD

**Issue by jfstephe** [aurelia/pal-nodejs#46]
**I'm submitting a bug report**

* **Library Version:**
2.0.0

**Please tell us about your environment:**
* **Operating System:**
Linux Ubuntu

* **Node Version:**
10.16.0

* **NPM Version:**
6.9.0

* **JSPM OR Webpack AND Version**
npm - using this on the server

* **Browser:**
N/A

* **Language:**
Typescript
4.1.5

**Current behavior:**
Security issue reported by snyk:
Run 'snyk test' and you get
```
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-AJV-584908] in ajv@6.10.2
    introduced by aurelia-pal-nodejs@2.0.0 > jsdom@15.2.1 > request@2.88.0 > har-validator@5.1.3 > ajv@6.10.2 and 1 other path(s)
  This issue was fixed in versions: 6.12.3
```

**Expected/desired behavior:**
No security issues

* **What is the expected behavior?**
No security issues

* **What is the motivation / use case for changing the behavior?**
Security

I *think* it may be a case of rebuilding the package and re-publishing but not 100%.

**Comment by jadrake75** [comment]
yes... they just need to update the package.json requires to a version like 16.7.0+

It no longer uses request which does not use the old version of uuid which has the vulnerability

**Comment by Xenope** [comment]
Any news about upgrading jsdom version? would be great to get rid of request-promise-native and har-validator.

**Comment by bigopon** [comment]
Thanks for the issue. Jsdom dep has now been upgraded to v19. And it's released as 3.0.0-rc.1
